### PR TITLE
Event Location and related models

### LINKED ISSUE #5: Event info with location
- Display event information
- No need for Luma anymore to reduce the amount of pages people need to visit
- Location data in Event

### PR DESCRIPTION
**Pull request by fikrialwan**
## What kind of change does this PR introduce?

Create function and UI to add event location when create event.

Event Location is available if the Category is In Person.

- [x] Create schema and seed data location categories
- [x] Create UI add event location
- [x] Create function to add event location
- [x] Show event location on event page detail 

## What is the current behavior?

- https://staging.bandungdev.com/events/2023-11-11
Screen Shot 2023-12-24 at 14 34 10
&nbsp;

-  https://staging.bandungdev.com/admin/events/clqj6drub0006nz0pgbk4g0jc
Screen Shot 2023-12-24 at 14 36 59
&nbsp;

## What is the new behavior?
Screen Shot 2023-12-29 at 15 43 24
new behaviour when create and update events
&nbsp;

Screen Shot 2023-12-29 at 23 47 37
Screen Shot 2023-12-29 at 23 50 15
Screen Shot 2023-12-29 at 23 51 10

new behaviour event page

&nbsp;
&nbsp;
Closes #5

### CODE REVIEW

File: prisma/data/location-categories.json
Comment (mhaidarhanif) [inline prisma/data/location-categories.json:1]: Can change Offline to In Person (IN_PERSON), like with Luma

File: prisma/schema.prisma
Comment (mhaidarhanif) [inline prisma/schema.prisma:1]: `link` here can be just `url`

Link would imply there are both `url`, `name`, `icon`, etc inside it.

File: prisma/data/location-categories.json
Comment (fikrialwan) [reply prisma/data/location-categories.json:1]: I've change Offline to In Person on this commit https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/62c4fe5878e65f6045a3eadd49c7f1e41bb540d9 and format code with Prettier on this commit https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/6ed31a587af32506a1c1f961da408ebf2af8920a

File: prisma/schema.prisma
Comment (fikrialwan) [reply prisma/schema.prisma:1]: i've changed link to be url on location schema on this commit https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/21c191bb9eb81cb4e71644e1f8e54cc97441c3ad

**Comment by mhaidarhanif** [comment]
@fikrialwan and @babdurrahman78 can use the same existing branch to work together, no need to create a separated branch.

**Comment by mhaidarhanif** [comment]
@fikrialwan @babdurrahman78 Once it's ready, let me know so we can review it. Also possible to review together in the usual video call again.

**Comment by fikrialwan** [comment]
@mhaidarhanif this pull request is ready now, for review together, I possible tonight

**Review by mhaidarhanif — CHANGES_REQUESTED** [review]
@fikrialwan @babdurrahman78 

- Fix import to use `~` such as `~/components`
- Remove organizerId requirement in model admin event
- Fix layout to edit the detail to be simpler in the middle
- Event Category radio button background color
- Event page to move Category to below Format, with just a text
- Rename Location (Offline) to Location (In Person)
- Show Maps URL below Location In Person

**Comment by fikrialwan** [comment]
@mhaidarhanif 
I've done fix all of the feedback
- Fix import to use ~ such as ~/components https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/41dd8e9201cb0d34366b88de72bf72aac77685c6
- Remove organizerId requirement in model admin event https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/9af66ed61a836b96272b73077aa31ba5deeff797
- Fix layout to edit the detail to be simpler in the middle https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/81f6ef857bd3e204560a111eb509fa57e96841de
- Event Category radio button background color  https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/32d9773fa358465903a0e5acb6e6ccd897c7fa6f
- Event page to move Category to below Format, with just a text https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/96586435aca1eab6504f2f2aea1d5fdb6f15e2a1
- Rename Location (Offline) to Location (In Person) https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/c54ee41c973a4ddf398454e7688d4f2cc69091bc
- Show Maps URL below Location In Person https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/35aea5d1c8e25c5f559451e9228b56ed43c90e22
- Fix render if location is null https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/ae004ebe3edec0d3abaf68ffd76de3ae74c17990
- Don't show Format and Category label if the data is unavailable https://github.com/bandungdevcom/bandungdev.com/pull/23/commits/04dd87c8da6bab20dcf41fbe88c516b015376e49

**Review by mhaidarhanif — APPROVED** [review]

**Pull request merged**